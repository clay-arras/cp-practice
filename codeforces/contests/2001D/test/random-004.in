1
15
17 9 5 7 9 17 5 3 17 6 8 13 2 13 2
