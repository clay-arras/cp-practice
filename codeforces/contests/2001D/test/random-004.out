9
17 5 7 9 3 6 8 2 13
